addons/mail/static/src/models/thread/thread.js

```diff
@@ -658,10 +658,15 @@ function factory(dependencies) {
         /**
          * Marks as read all needaction messages with this thread as origin.
          */
-        async markNeedactionMessagesAsOriginThreadAsRead() {
-            await this.async(() =>
-                this.env.models['mail.message'].markAsRead(this.needactionMessagesAsOriginThread)
-            );
+        async markMessagesAsOriginThreadAsRead() {
+            if (Number.isInteger(this.id)) {
```

Comment:
I don't know what this is but it doesn't look right. Is it for temporary thread (when doing create?) and if so, the isLoaded condition should prevent this from being called in stupid cases.

Reply:
Sometimes `this.id == 'inbox'`.
I'll try to remove it when the `isLoaded` condition is working

Reply:
Probably condition is better on model !== mail.box then